server/db/models/user.js

```diff
@@ -26,6 +46,21 @@ const User = db.define('user', {
   },
   googleId: {
     type: Sequelize.STRING
+  },
+  Address: {
+    type: Sequelize.STRING
+  },
+  Account: {
+    type: Sequelize.INTEGER,
+    unique: true,
+    allowNull: false
+  },
+  PhoneNumber: {
+    type: Sequelize.INTEGER
+  },
+  orderHistory: {
```

Comment:
I think we're better off leaving this blank for now rather than putting something in that might cause errors